src/frontend/components/Felleskomponenter/BlokkerTilbakeKnappModal/BlokkerTilbakeKnappModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+
+import { useIntl } from 'react-intl';
+import { Prompt } from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import { Flatknapp } from 'nav-frontend-knapper';
+import Modal from 'nav-frontend-modal';
+import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
+
+import Knappelenke from '@navikt/sif-common-core/lib/components/knappelenke/Knappelenke';
+
+import SpråkTekst from '../SpråkTekst/SpråkTekst';
+
+const StyledModal = styled(Modal)`
+    max-width: 600px;
```

Comment:
Nit: tror vi som regel har brukt rem andre steder i kode, så kan vurdere å bruke det i stedet 😊

Reply:
Ja, veldig fint å bruke rem, for da skalerer den også riktig dersom man endrer font-størrelse i browser 😊